manifests/v1alpha3/katib-controller/trialTemplateConfigmapLabeled.yaml

```diff
@@ -0,0 +1,95 @@
+apiVersion: v1
```

Comment:
I am reluctant to have them in v1alpha3 itself as this would confuse users. What do you think?

Reply:
This change will not affect old trail template and controller. 
User just can't see old Trial Templates and Submit Experiment by Parameters from the UI, using not labeled trial templates.
Other functionality in the UI will work, so they can submit Experiment using yaml file and monitor Experiments

You think that it is better not include this trial template to manifests in v1alpha3?